File: AZ-204_Implement_Caching_For_Solutions/code/interact.py
import redis

# Connect to the Redis Cache
redis_cache = redis.Redis(host='<your_redis_host>', port=<your_redis_port>, password='<your_redis_password>')

# Set a key-value pair in the cache
redis_cache.set('my_key', 'my_value')

# Get the value for a given key from the cache
value = redis_cache.get('my_key')
print(value)

# Delete a key from the cache
redis_cache.delete('my_key')

File: AZ-204_Implement_Caching_For_Solutions/code/interact.cs
using StackExchange.Redis;
using System;

class Program
{
    static void Main()
    {
        // Connect to the Redis cache
        ConnectionMultiplexer redis = ConnectionMultiplexer.Connect("<your-redis-connection-string>");

        // Get a reference to the cache database
        IDatabase cache = redis.GetDatabase();

        // Store a value in the cache
        cache.StringSet("mykey", "myvalue");

        // Retrieve the value from the cache
        string value = cache.StringGet("mykey");

        Console.WriteLine("Value from cache: " + value);

        // Remove the value from the cache
        cache.KeyDelete("mykey");

        // Close the connection to the Redis cache
        redis.Close();
    }
}